app/src/renderer/components/staking/PageStaking.vue

```diff
@@ -7,11 +7,15 @@ tm-page(title='Staking')
       i.search.material-icons search
 
   modal-search(type="delegates" v-if="somethingToSearch")
-
+  .delegates-tabs
+    .tab(v-for="(tab, i) in tabs",
+      :key="'tab-' + i",
+      :class="{'tab-selected': i === tabIndex}",
+      @click="tabIndex = 1") {{tab}}
   .delegates-container
-    tm-data-loading(v-if="delegates.loading && delegates.delegates.length === 0")
-    tm-data-empty(v-else-if="delegates.delegates.length === 0")
-    data-empty-search(v-else-if="sortedFilteredEnrichedDelegates.length === 0")
+    tm-data-loading(v-if="delegates.loading && sortedFilteredEnrichedDelegates.length === 0")
+    tm-data-empty(v-else-if="!delegates.loading && delegates.delegates.length === 0")
```

Comment:
should `delegates.delegates.length` be `sortedFilteredEnrichedDelegates.length`?

Reply:
smells like a refactor (one day).

`delegates.delegates.delegates.delegates.delegates.goose`

🦆 🦆 🦆 🦆 🦆

Reply:
didn't we have the discussion before? ;)